android/assets/i18n/messages.properties

```diff
@@ -84,37 +84,37 @@ multiplayer.server-list.last-player-seen.a-long-time-ago=No activity for a long
 multiplayer.final-scores.winner=Winner
 multiplayer.final-scores.others=Other
 multiplayer.final-scores.winner.sole-survivor-1=You win. You are the sole survivor.\nYou have the highest score. Nobody can catch you.
-multiplayer.final-scores.winner.sole-survivor-2=You win. You outlasted your challengers.\nYour score is the best.
+multiplayer.final-scores.winner.sole-survivor-2=You win. You outlasted your challengers.\nYour score is the highest.
```

Comment:
This one I'd rather leave as "best", because it is supposed to be a bit more gloating rather than informative. Although "Your score is the highest" is more technically correct, I still feel "Your score is the best" it is a little bit more fun and light hearted.